lib/oref0-setup/report.json

```diff
@@ -18,7 +18,7 @@
       "use": "shell",
       "reporter": "JSON",
       "device": "ns",
-      "remainder": "-6hours",
+      "remainder": "-18hours",
```

Comment:
This can be set back to `-4hours`, once a Nightscout with this PR is used https://github.com/nightscout/cgm-remote-monitor/pull/2763 . In fact it depends what we expect of the maximum temp target time.

I think it's fine to stick to `-18hours` for the 0.5.3 release, but maybe we should urge people in the docs to upgrade their Nightscout once the next cgm-remote-monitor version (with my PR included) is released.

Reply:
I'm fine with encouraging people to run the latest NS, but is there any real downside to downloading extra carbhistory or temptargets?  As far as I can tell oref0 correctly ignores any carb treatments older than 6h, so there doesn't seem to be any functionality change or performance hit from downloading extra.  I'm not sure whether the longer download would allow for temp targets longer than 6 hours, but I'm OK with that either way...